test/auth/auth-manager-test.js

```diff
@@ -0,0 +1,129 @@
+const mockgoose = require('mockgoose');
+const mongoose = require('mongoose');
+const bluebird = require('bluebird');
+const chai = require('chai');
+const chaiMoment = require('chai-moment');
+const jwt = require('jsonwebtoken');
+const authManager = require('../../src/auth/auth-manager');
+const authModels = require('../../src/auth/auth-models');
+const userManager = require('../../src/users/user-manager');
```

Comment:
I can remove the `user-manager`, but I will still need to use `user-models`.

`auth` and `users` are tightly coupled.